primitives/version/versions.go

```diff
@@ -20,23 +20,64 @@
 
 package version
 
+import (
+	"github.com/berachain/beacon-kit/primitives/bytes"
+	"github.com/berachain/beacon-kit/primitives/common"
+)
+
 const (
-	// Phase0 is the first version of the Beacon Chain.
-	Phase0 uint32 = 0
-	// Altair is the first hardfork of the Beacon Chain.
-	Altair uint32 = 1
-	// Bellatrix is the second hardfork of the Beacon Chain.
-	Bellatrix uint32 = 2
-	// Capella is the third hardfork of the Beacon Chain.
-	Capella uint32 = 3
-	// Deneb is the first version of Deneb, used for genesis of Berachain mainnet.
-	Deneb uint32 = 4
-	// Deneb1 is the first hardfork of Deneb on Berachain mainnet (TBD if used).
-	// There may also be Deneb2, Deneb3, etc. hardforks.
-	Deneb1 uint32 = 260
-	// Electra is the first version of Electra on Berachain mainnet.
-	Electra uint32 = 5
-	// Electra1 is the first hardfork of Electra on Berachain mainnet (TBD if used).
-	// There may also be Electra2, Electra3, etc. hardforks.
-	Electra1 uint32 = 261
+	// phase0 is the first version of the Beacon Chain.
+	phase0 uint32 = 0
+	// altair is the first hardfork of the Beacon Chain.
+	altair uint32 = 1
+	// bellatrix is the second hardfork of the Beacon Chain.
+	bellatrix uint32 = 2
+	// capella is the third hardfork of the Beacon Chain.
+	capella uint32 = 3
+	// deneb is the first version of Deneb, used for genesis of Berachain mainnet.
+	deneb uint32 = 4
+	// deneb1 is the first hardfork of Deneb on Berachain mainnet. LittleEndian of {4, 1, 0, 0}.
+	deneb1 uint32 = 260
+	// electra is the first version of Electra on Berachain mainnet.
+	electra uint32 = 5
```

Comment:
nit: probably excessive but we could:
- explicitly define these as `var <fork_name> []byte = <their byte representation>`
- copy the byte slices in the getter to avoid any client modifying them

It would make clearer that ordering of version follows the byte representation, not the numerical one

Reply:
keeping them as vars but private fine with me but then we have global vars, indifferent either way

Reply:
done, kept as vars